fr-French/Localizable.strings

```diff
@@ -0,0 +1,298 @@
+"activate" = "activer";
+"activated" = "activé";
+"deactivate" = "désactiver";
+"deactivated" = "désactivé";
+"cancel" = "Annuler";
+"close" = "Ferme";
+"confirm" = "Confirmer";
+"copy" = "Copier";
+"copied" = "Copié";
+"create" = "Créer";
+"created" = "Créé";
+"delete" = "Supprimer";
+"done" = "Fini";
+"modify" = "Modifier";
+"retry" = "Réessayer";
+"save" = "Sauvegarder";
+// Left menu
+"alias" = "alias de courriels";
+"aliases" = "alias de courriels";
+"mailbox" = "mailbox (boîte de réception)";
+"mailboxes" = "mailbox (boîtes de réception)";
+"settings" = "Réglages";
+"about" = "A propos";
+"rate_us" = "Voter";
+"sign_out" = "Se déconnecter";
+"sign_out_confirmation" = "Vous allez vous déconnecter";
+"yes_sign_me_out" = "Oui";
+// Alias
+"forward" = "transfert";
+"forwards" = "transferts";
+"block" = "bloqué";
+"blocks" = "bloqués";
+"reply" = "réponse";
+"replies" = "réponses";
+"send_email" = "Envoyer email";
+"deleted_alias" = "Supprimer alias \"%@\"";
+// Alias detail
+"edit_mailboxes" = "Éditer mailbox";
+"add_name" = "Ajouter nom";
+"add_name_for_alias" = "Ajouter un nom pour un alias";
+"edit_name" = "Éditer nom";
+"edit_name_for_alias" = "Éditer nom pour un alias";
+"no_display_name" = "Nom caché";
+"add_note" = "Ajouter un notre";
+"add_note_for_alias" = "Ajouter un note pour un alias";
+"no_note" = "Pas de note";
+"edit_note" = "Éditer note";
+"edit_note_for_alias" = "Éditer note pour un alias";
+"email_handled" = "Email traité";
+"emails_handled" = "Emails traités";
+"email_forwarded" = "Email transféré";
+"emails_forwarded" = "Emails transférés";
+"email_replied" = "Email répondu";
+"emails_replied" = "Emails répondus";
+"email_blocked" = "Email bloqué";
+"emails_blocked" = "Emails bloqués";
+// Send email
+"compose_and_send_email" = "Composer et envoyer email";
+"from" = "De";
+"to" = "À";
+"copy_reverse_alias_with_display_name" = "Copier reverse-alias (avec nom)";
+"copy_reverse_alias_without_display_name" = "Copier reverse-alias (sans nom)";
+"begin_composing_with_default_email" = "Composer avec l'application mail par défaut";
+// Alias list
+"all" = "Tout";
+"active" = "Activé";
+"inactive" = "Inactivé";
+"delete_alias_confirmation" = "🛑 Ceux qui vous avez contacté par cet alias ne peuvent plus vous contacter. L'opération est irréversible, veuillez confirmer.";
+// Create alias
+"create_alias" = "Créer un Alias";
+"create_alias_placebolder" = "Par example: groupon, newsletters";
+"create_alias_name_requirements_explication" = "Seuls les minuscule, chiffre, - et _ sont supportés.";
+"create_alias_mailboxes_explication" = "Alias appartient à ce(s) mailbox(es)";
+"name" = "Nom";
+"create_alias_name_explication" = "Le nom qui s'affiche quand vous répondez ou envoyez un mail (optionnel)";
```

Comment:
I think it should be "facultatif" instead of "optionnel"

Reply:
Both are good. They are synonyms.